ttl/cache/ttlstatus.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/tidb/ttl/session"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+// JobStatus represents the current status of a job
+type JobStatus string
+
+const (
+	// JobStatusWaiting means the job hasn't started
+	JobStatusWaiting JobStatus = "waiting"
+	// JobStatusRunning means this job is running
+	JobStatusRunning = "running"
+	// JobStatusCancelling means this job is being canceled, but not canceled yet
+	JobStatusCancelling = "cancelling"
+	// JobStatusCancelled means this job has been canceled successfully
+	JobStatusCancelled = "cancelled"
+	// JobStatusError means this job is in error status
+	JobStatusError = "error"
+)
+
+const selectFromTTLTableStatus = "SELECT table_id,parent_table_id,table_statistics,last_job_id,last_job_start_time,last_job_finish_time,last_job_ttl_expire,last_job_summary,current_job_id,current_job_owner_id,current_job_owner_addr,current_job_owner_hb_time,current_job_start_time,current_job_ttl_expire,current_job_state,current_job_status,current_job_status_update_time FROM mysql.tidb_ttl_table_status"
+
+// TableStatus contains the corresponding information in the system table `mysql.tidb_ttl_table_status`
+type TableStatus struct {
+	TableID       int64
+	ParentTableID int64
+
+	TableStatistics string
+
+	LastJobID         string
+	LastJobStartTime  types.Time
```

Comment:
Maybe go time instead of `types.Time` is better?

Reply:
Yes! Then I don't need to convert to go time everytime I used these fields :+1: .

It will use the time zone in session vars.

Reply:
Fixed.